3D Chess/Assets/Scripts/PieceEntity.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Realms;
+using UnityEngine;
+
+public class PieceEntity : RealmObject
+{
+    public int Type { get; set; }
+
+    public float PositionX { get; set; }
+    public float PositionY { get; set; }
+    public float PositionZ { get; set; }
+
+    public PieceEntity()
```

Comment:
This ctor can be private.

Reply:
I actually thought, constructors and fields need to be public. 🙈 

That's really great since having an empty default constructor just because we have to (without using it) was a bit of a nuisance. 👍 

Changed with 6d79643.

Reply:
Yeah, everything can be private, which is quite convenient to force the public API consumer to always provide the required arguments.